app.ts

```diff
@@ -1,10 +1,10 @@
-const log = require('./lib/logger');
+import { log } from "./lib/logger";
 const expressServer = require('./lib/express');
 const getLag = require('./lib/get_lag');
 const saveLag = require('./lib/save_lag');
 const fetchVersion = require('./lib/fetch_version');
 
-function saveDateAndScheduleJob (lagData) {
+function saveDateAndScheduleJob (lagData: any) {
```

Comment:
`any`?!?!!

Reply:
haha yup. just to get things to compile. I definitely plan to come back and use some real types now. This should give me the confidence to make more changes.